src/pages/archive/index.js

```diff
@@ -26,7 +32,7 @@ Archive.propTypes = {
 export const meetingsQuery = graphql`
   query MeetingsQuery {
     allMarkdownRemark(
-      filter: { fileAbsolutePath: {regex : "\\/meetings/"} },
+      filter:{fields:{type:{eq:"meeting"}}},
       sort: { fields: [frontmatter___date], order: DESC }
```

Comment:
to start at parity with the old site, im also hoping to paginate this list (once we have enough meetings moved over to have more than one page of data). have any good docs to point me at for that?

Reply:
@travi general idea is to chunk (at the createPages level) and then create the path like archive/2, archive/3, etc. depending on how you chunk. So archive becomes a template, rather than a page if that makes sense!

Reply:
i think so, but ill have to dig into an implementation to fully get my head around it. i imagine its pretty straight forward once i get around to it